contrib/build-linux/appimage/_build.sh

```diff
@@ -161,7 +161,7 @@ info "Finalizing AppDir"
     # note: temporarily move PyQt5 out of the way so
     # we don't try to bundle its system dependencies.
     mv "$APPDIR/usr/lib/python3.6/site-packages/PyQt5" "$BUILDDIR"
-    copy_deps; copy_deps; copy_deps
+    copy_deps
```

Comment:
Lol. Why?!?!? :)

Reply:
The comment of pkg2appimage copy_deps actually says it can be beneficial to do it multiple times, but in our case it isn't:

https://github.com/AppImage/pkg2appimage/blob/eb8f3acdd9f11ab19b78f5cb15daa772367daf15/functions.sh#L93